pkgs/by-name/ug/ugm/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
+}:
+
+buildGoModule rec {
+  pname = "ugm";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "ariasmn";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Co8JN0WEc1I08My9m7iyAshtEO4aszN8/sCvoGFJv2A=";
+  };
+
+  vendorHash = "sha256-34D9fQnmKnOyUqshduLmFiVgcVKi7mDKBs3X5ZQxsuw=";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  # Fix unaligned table when running this program under a CJK environment
```

Comment:
Is this nessarily needed for all users?

Reply:
This environment variable is necessary for those who work with locale set to zh_CN.UTF-8 or jp_JP.UTF-8, e.g. to prevent broken table outputs. For users who aren't in such environments, this variable is likely to have no impact, since CJK characters rarely appear in their environment. In addition, users and group information almost never contains CJK characters.

Reply:
lgtm